pandas/tests/groupby/test_groupby.py

```diff
@@ -508,30 +507,30 @@ def test_frame_multi_key_function_list():
 
 
```

Comment:
we could prob split the panel tests to a separate file (and just ignore warnings there), maybe make an issue